src/System.Text.Encoding.CodePages/src/System/Text/EncodingTable.Data.cs

```diff
@@ -1250,6 +1250,7 @@ internal static partial class EncodingTable
             28599, // iso-8859-9
             28603, // iso-8859-13
             28605, // iso-8859-15
+            29001, // x-europa
```

Comment:
I did edit the file by hand (not having your tool), although I actually used Excel to do the math on the indices.